src/_help/customization-options/logo-meta-images.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Logo and Meta images
+---
+
+- TOC
+{:toc}
+
+## Logo and favicon
+
+You can upload your own logo and favicon on Bump.sh to customize your documentation with your brand or project colors.
+
+From the "Customize UI" tab in the documentation settings, you can add your own logo and favicon.
+
+![](/images/help/logo-favicon.png)
+
+To provide the best image quality, it is recommended to adhere to the following dimensions for your logo: 140px*36px.
+
+
+## Meta images
+
+A "meta image" typically refers to an image that is used as a preview image when a web page is shared on social media platforms and can be displayed in search engine results. You may have already heard it referred to as a "social media image." Meta images help make the links posted on social media platforms more attractive, especially when shared or displayed in posts.
```

Comment:
I think we should start by mentioning that a meta image is generated by Bump.sh, by default. And change the order of the 2 sections.

Customizing the meta image is not something a lot of our customers do.